products/rubymine.md

```diff
@@ -0,0 +1,42 @@
+---
+title: RubyMine Plugin Development
+---
+
+## Introduction
+RubyMine is an IntelliJ Platform-based product.
+Plugin projects for RubyMine can be developed using IntelliJ IDEA with the `gradle-intellij-plugin`.
+
+## Configuring Plugin Projects Targeting RubyMine
+The configuration of RubyMine plugin projects follows the methods described in [Configuring Plugin Projects using the IntelliJ IDEA Product Attribute](dev_alternate_products.md#configuring-plugin-projects-using-the-intellij-idea-product-attribute), and [Configuring the plugin.xml File](dev_alternate_products.md#configuring-pluginxml).
+
+The table below summarizes the `gradle-intellij-plugin` attributes to set in the `build.gradle` file for a RubyMine plugin project:
+
+| `gradle-intellij-plugin` Attribute | <br>Attribute Value |
```

Comment:
`<br>` = ?

Reply:
It's just formatting - a poor man's way to set the "Attribute Value" to bottom justify.